Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -37,40 +37,41 @@ public struct WebAuthnManager {
     /// - Parameters:
     ///   - config: The configuration to use for this manager.
     ///   - challengeGenerator: The challenge generator to use for this manager. Defaults to a live generator.
-    public init(config: WebAuthnConfig, challengeGenerator: ChallengeGenerator = .live) {
+    public init(config: Config, challengeGenerator: ChallengeGenerator = .live) {
         self.config = config
         self.challengeGenerator = challengeGenerator
     }
 
-    /// Generate a new set of registration data to be sent to the client and authenticator.
+    /// Generate a new set of registration data to be sent to the client.
     ///
+    /// This method will use the Relying Party information from the WebAuthnManager's config  to create ``PublicKeyCredentialCreationOptions``
     /// - Parameters:
     ///   - user: The user to register.
-    ///   - attestation: The level of attestation to be provided by the authenticator.
+    ///   - timeoutInSeconds: How long the browser should give the user to choose an authenticator. This value
+    ///     is a *hint* and may be ignored by the browser. Defaults to 60 seconds.
+    ///   - attestation: The Relying Party's preference regarding attestation. Defaults to `.none`.
     ///   - publicKeyCredentialParameters: A list of public key algorithms the Relying Party chooses to restrict
     ///     support to. Defaults to all supported algorithms.
     /// - Returns: Registration options ready for the browser.
     public func beginRegistration(
-        user: WebAuthnUser,
-        timeout: TimeInterval = 60000,
+        user: PublicKeyCredentialUserEntity,
+        timeoutInSeconds: TimeInterval? = 3600,
```

Comment:
Hi @marius-se - it looks like 60 mins, not 60 seconds as described in line 51

Reply:
Yep this looks like a bug - feel free to submit a PR if you'd like!

Reply:
Here we go: #65